content/02.main-text.md

```diff
@@ -111,6 +111,8 @@ With Manubot, manuscripts are written as plain-text Markdown files, which is wel
 The Markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
 In addition, Manubot relies on extensions from [Pandoc Markdown](https://pandoc.org/MANUAL.html#pandocs-markdown) to enable citations, tables, captions, and equations specified using the popular TeX math syntax.
 
+## Citation by identifier
```

Comment:
This can be a `###` header.  In #183 I also renamed this section to `Manubot features` to help differentiate it from the `Manubot contribution workflow` section.